arm/Microsoft.Compute/virtualMachines/deploy.bicep

```diff
@@ -3,6 +3,11 @@
 param name string = take(toLower(uniqueString(resourceGroup().name)), 10)
 
 @description('Optional. Specifies whether the computer names should be transformed. The transformation is performed on all computer names. Available transformations are \'none\' (Default), \'uppercase\' and \'lowercase\'.')
+@allowed([
```

Comment:
Wondering about the general value of this parameter

Reply:
Fair point. But I wouldn't remove it in this PR I guess. Might have been a specific requirement for the original implementation of the module that should not be carried over. Anyhow something to cover separatly if you agree